client/src/pages/Landing.js

```diff
@@ -41,29 +41,30 @@ class LandingPage extends Component {
   render() {
     const { classes } = this.props;
     return (
-      <div className={classes.landingContainer}>
-        <Typography>{this.state.welcomeMessage}</Typography>
-        {this.state.step >= 1 && (
-          <React.Fragment>
-            <Link to="/ping">Step 2: Click here </Link>
-            <Route
-              path="/ping"
-              render={props => {
-                return (
-                  <Ping
-                    {...props}
-                    incrementStep={this.incrementStep}
-                    step={this.state.step}
-                  />
-                );
-              }}
-            />
-          </React.Fragment>
-        )}
-        {this.state.step >= 3 && (
-          <Typography>All done! Now go make a pull request!</Typography>
-        )}
-      </div>
+      // <div className={classes.landingContainer}>
```

Comment:
you can feel free to remove this entire landing page if it's not dependant anymore!